### PR TITLE
Rewrote `notification-service` to use RPC URLs

### PR DESCRIPTION
**Pull request by zoey-kaiser**
Contributes to #330

I saw that in #330 we did not rewrite the `notification-service` to use RPC URLs, so I opened this PR to fix this!

Checklist:
- [x] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [x] issue checkboxes are all addressed
- [x] manually checked my feature / not applicable
- [x] wrote tests / not applicable
- [x] attached screenshots / not applicable

### CODE REVIEW

**Comment by zoey-kaiser** [comment]
From my testing it seems to work well, I only ran into one small issue:

When using the `mainnet` from infura, it returns the network name `homestead`. This obviously messes up some of our internal logic, as we always refer to the network as the `mainnet`. What is the best way to address this issue? If it is only this one network, I could imagine just referring to it as homestead in the `notification-service`, as the service runs separately from our core and other projects.

**Comment by zoey-kaiser** [comment]
> When using the `mainnet` from infura, it returns the network name `homestead`. This obviously messes up some of our internal logic, as we always refer to the network as the `mainnet`. What is the best way to address this issue? If it is only this one network, I could imagine just referring to it as homestead in the `notification-service`, as the service runs separately from our core and other projects.

I renamed `mainnet` to ` homestead` in this project to ensure correct functionality.

File: notification-service/README.md [hunk 1]
```diff
@@ -27,9 +27,8 @@ $ npm run start
 
 Note: env variables are accessible via the `secret` command under `auction-ui/${environment}/notification-service`.
 
-- `INFURA_PROJECT_ID`: (required) [infura](https://infura.io/) project id (can be found in: dashboard -> ethereum ->
+- `WS_URL`: (required) Ethereum Websocket url used for fetching data from the blockchain
```
Comment (valiafetisov) [inline notification-service/README.md:30]: Suggested naming would be `RPC_WEBSOCKET_URL`

File: notification-service/src/constants/NETWORKS.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const NETWORKS: Record<string, { network: string; etherscanURL: string; etherscanAPI: string }> = {
-    mainnet: {
-        network: 'mainnet',
+    homestead: {
+        network: 'homestead',
```
Comment (valiafetisov) [inline notification-service/src/constants/NETWORKS.ts:3]: > When using the `mainnet` from infura, it returns the network name `homestead`

What do you mean by `it returns`?

**Pull request closed without merging**